system_metrics_collector/test/system_metrics_collector/test_linux_process_cpu_measurement_node.cpp

```diff
@@ -119,8 +119,7 @@ class LinuxProcessCpuMeasurementTestFixture : public ::testing::Test
 
     ASSERT_FALSE(test_measure_linux_process_cpu_->IsStarted());
 
-    const moving_average_statistics::StatisticData data =
-      test_measure_linux_process_cpu_->GetStatisticsResults();
+    const StatisticData data = test_measure_linux_process_cpu_->GetStatisticsResults();
```

Comment:
```suggestion
    const auto data = test_measure_linux_process_cpu_->GetStatisticsResults();
```

Reply:
Done.

Reply:
Sorry. I missed this, but I didn't want to wait for CI to re-run again. I also think this doesn't reduce that many characters on this line of code, so it would have been an insignificant change.